ktor-server/ktor-server-jetty/jvm/test/io/ktor/tests/server/jetty/JettyServletApplicationEngine.kt

```diff
@@ -7,42 +7,24 @@ package io.ktor.tests.server.jetty
 import io.ktor.server.engine.*
 import io.ktor.server.jetty.*
 import io.ktor.server.servlet.*
-import io.ktor.server.testing.*
 import org.eclipse.jetty.servlet.*
-import org.junit.*
 import javax.servlet.*
 
-class JettyAsyncServletContainerEngineTest :
-    EngineTestSuite<JettyApplicationEngineBase, JettyApplicationEngineBase.Configuration>(
-        Servlet(async = true)
-    )
-
-class JettyBlockingServletContainerEngineTest :
-    EngineTestSuite<JettyApplicationEngineBase, JettyApplicationEngineBase.Configuration>(
-        Servlet(async = false)
-    ) {
-    @Ignore
-    override fun testUpgrade() {
-    }
-}
-
-// the factory and engine are only suitable for testing
-// you shouldn't use it for production code
-
-private class Servlet(private val async: Boolean) :
+/**
+ * The factory and engine are only suitable for testing. You shouldn't use it for production code.
+ */
```

Comment:
Is there a way to use it if it's internal in `*.test.*` package?

Reply:
Upd: Well, I get the point. I think we should make some annotation for that, like `@TestOnly` and check that it's being used only in test module